CHANGELOG.md

```diff
@@ -3,6 +3,73 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.2.0
+
+_Dec 9, 2021_
+
+A big thanks to the 5 contributors who made this release possible. Here are some highlights ✨:
+
+### `@mui/x-data-grid@v5.2.0` / `@mui/x-data-grid-pro@v5.2.0`
+
+- 🚀 Introduce the [column pinning](https://mui.com/components/data-grid/columns/#column-pinning) feature (#2946) @m4theushw
+
+  <img src="https://user-images.githubusercontent.com/42154031/145425635-b6314fbe-2f1e-4b73-908f-33ee1fda20c7.gif" width="964" height="657">
+
+- 🔥 Add ability to disable export options (#3270) @alexfauquette
+
+  You can disable either export options by setting `disableToolbarButton` to `true`.
+
+  ```tsx
+  <GridToolbarExport csvOptions={{ disableToolbarButton: true }} />
+  ```
+
+  ```tsx
+  <GridToolbarExport printOptions={{ disableToolbarButton: true }} />
+  ```
+
+- 🙈 Add `hideDescendantCount` option to Tree Data (#3368) @flaviendelangle
```

Comment:
Should we be more precise ?

```suggestion
- 🙈 Add a new option to hide the amount of descendant on the grouping cells of the Tree Data (#3368) @flaviendelangle
  
  ```tsx
  <DataGridPro
    treeData
    rows={rows}
    columns={columns}
    groupingColDef={{ hideDescendantCount }}
  />
  ```
```